jenkins/freestyle/disable-old-workers.groovy

```diff
@@ -0,0 +1,22 @@
+// This script is built for the script console in Jenkins
```

Comment:
nit: include a link (https://build.testeng.edx.org/script) so that it's easier for the user to navigate there without looking it up. e.g.